core/src/main/resources/lib/layout/ionicon.jelly

```diff
@@ -0,0 +1,18 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:d="jelly:define">
+    <st:documentation>
+        Provides ionicon icons
+        <st:attribute name="name" use="required">
+            The name of the icon
+        </st:attribute>
+        <st:attribute name="title">
+            Title to use for the icon
+        </st:attribute>
+    </st:documentation>
+
+    <j:invokeStatic var="icon" className="org.jenkins.ui.icon.IconSet" method="getIonicon">
+        <j:arg value="${name}"/>
+        <j:arg value="${title}"/>
+    </j:invokeStatic>
+    <j:out value="${icon}" />
```

Comment:
This looks somehow strange that we need to invoke sever code to render an image. I'm using a SVG sprite to render Font Awesome icons. 

How can I add an additional `class` to the icon? E.g., to change color or size? 

I used that approach in the font-awesome plugin:
https://github.com/jenkinsci/font-awesome-api-plugin/blob/master/src/main/resources/font-awesome/svg-icon.jelly

Reply:
it's explained in the description, it's to avoid having to modify them into sprites.

Reply:
Yep! The parent element would manage the styling for the icon, although modifier classes should continue to work.